yarn.lock

```diff
@@ -4067,8 +4067,10 @@ rollup@2.35.1:
   optionalDependencies:
     fsevents "~2.1.2"
 
-"rrule@file:../rrule":
+"rrule@\rrule":
```

Comment:
hmm, this was pointing at a local file previously. It was compiling on my machine. Let me revert this change, but I think a version should be pinned instead? Or a git hash if you have a fork?

Reply:
Linting is failing because it can't install rrule

Reply:
Here's the fix I am imaging, however it's not quite working yet. Builds seem to work fine, but tests complain that it can't find rrule. I'll take another look this evening.

https://github.com/tgrosinger/slated-obsidian/pull/35

```
$ ./yarn test 
yarn run v1.22.5
$ jest
 FAIL  tests/task-line.test.ts
  ● Test suite failed to run

    src/task-line.ts:15:34 - error TS2307: Cannot find module 'rrule' or its corresponding type declarations.

    15 import RRule, { Frequency } from 'rrule';
```

Reply:
It should be fixed now.